### ISSUE TITLE
Allow the use of dynamic style attributes

### ISSUE THREAD

**Issue by swelham** [ember-template-lint/ember-template-lint#555]
It would be nice if we could ignore the `inline-styles` rule when the style is being set dynamically at runtime, i.e. `<SomeElement style={{style}}>`. When runtime calculations are required, for example to correctly size an image to a fixed aspect ratio, the template linter prevents the use of a tagless component.

To implement a basic example we would currently have to implement our component like so.

```javascript
export default Component.extend({
  classNames: ['relative'],
  attributeBindings: ['style'],
  style: computed('ratio', function() {...})
});
```

However it would be more desired to keep the attributes and classes defined within the template, allowing us to do the following.

```javascript
export default Component.extend({
  tagName: '',
  style: computed('ratio', function() {...})
});
```

```hbs
<div class="relative" style={{style}}>
  {{yield}}
</div>
```

**Comment by MelSumner** [comment]
I'm curious- is there a reason the standard could be followed? 

- depending on the scenario, apply a specific class
- if it's a viewport issue, then have media queries in the CSS 

Inline styling just makes for really messy code. I'd rather someone turn off the rule completely then for us to try to rationalize if/when it should be allowed. Just my $.02 though!

**Comment by lifeart** [comment]
Closing issue as we could disable rule in-place once needed
